docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Database Access with Self-Hosted MongoDB
+description: How to configure Teleport Database Access with self-hosted MongoDB
+---
+
+# Self-Hosted MongoDB
+
+<Admonition type="note" title="Supported versions">
+  Teleport Database Access supports MongoDB 3.6 and higher. Older versions have
+  not been tested and are not guaranteed to work. MongoDB 3.6 was released in
+  November 2017 and reached EOL in April 2021 so if you're still using an older
+  version, consider upgrading.
+</Admonition>
+
+## Create Certificate/Key Pair
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+Create the secrets:
+
+<Tabs>
+  <TabItem label="Standalone">
+  When connecting to standalone MongoDB, sign certificate for the hostname over
+  which Teleport will be connecting to it.
+
+  For example, if your MongoDB server is accessible at `mongo.example.com`
+  hostname, run:
+
+  ```bash
+  $ tctl auth sign --format=mongodb --host=mongo.example.com --out=mongo --ttl=8760h
```

Comment:
Double-checking: we're using `8760h` here but `8192h` elsewhere in this PR. Is there one value we should use? Are these values specific to one of the DB's but not the others?